pyro/optim/lr_scheduler.py

```diff
@@ -27,17 +27,11 @@ def __init__(self, scheduler_constructor, optim_args):
         # kwargs for the torch optimizer
         optim_kwargs = optim_args.pop('optim_args')
         self.kwargs = optim_args
-        # current epoch
-        self.epoch = None
```

Comment:
@jpchen - Thanks for fixing the test. Do we not need to hold the current epoch in `self.epoch` anymore? What was the change on the PyTorch side?

Reply:
the way `get_lr()` was implemented for these schedulers. also the initialization of `last_epoch` attribute though that doesnt affect this.  every time `svi.step()` is the scheduler will take a step now.